### PR TITLE
Dragos C2C (Source)

### PR DESCRIPTION
**Pull request by amee-sumo**
## Purpose of this pull request

This pull request is to create a new doc on Dragos.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-593

### CODE REVIEW

**Comment by jpipkin1** [comment]
@amee-sumo, I changed the name of the release note file to `2024-12-13-collection.md` so it wouldn't result in a merge conflict. You'll have to change the file name again to the date it will actually be published.

File: docs/integrations/product-list/product-list-a-l.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ For descriptions of the different types of integrations Sumo Logic offers, see [
 | <img src={useBaseUrl('img/integrations/misc/doppel-logo.png')} alt="Thumbnail icon" width="75"/> | [Doppel](https://www.doppel.com/) | Partner integration: [Doppel Vision](https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/DoppelVision) |
 |  <img src={useBaseUrl('img/integrations/misc/doppler-logo.png')} alt="Thumbnail icon" width="50"/>   | [Doppler](https://www.doppler.com/)  | Partner integration:	[Doppler](https://docs.doppler.com/docs/sumologic) |
 |  <img src={useBaseUrl('img/platform-services/automation-service/app-central/logos/downdetector.png')} alt="Thumbnail icon" width="100"/> | [Downdetector](https://downdetector.com/) | Automation integration: [Downdetector](/docs/platform-services/automation-service/app-central/integrations/downdetector/) |
+|  <img src={useBaseUrl('img/send-data/dragos-logo.png')} alt="Thumbnail icon" width="60"/> | [Dragos](https://www.dragos.com//) | Collector: [Dragos](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source) |
```
Comment (jpipkin1) [inline docs/integrations/product-list/product-list-a-l.md:200]:
```suggestion
|  <img src={useBaseUrl('img/send-data/dragos-logo.png')} alt="Thumbnail icon" width="60"/> | [Dragos](https://www.dragos.com//) | Collector: [Dragos Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source) |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+Follow the instructions below to generate the API ID and API Secret:
+
+1. Open the Dragos platform and navigate to the **Admin** > **User**.
+1. Click **ADD NEW API KEY** under the **API Keys** section.
+1. Enter the name of the API Key in the **Name** field and then click **GENERATE KEY**.
+1. Copy the **API ID** and **API Secret**.
+
+### Source configuration
+
+When you create a Dragos Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:52]:
```suggestion
When you create a Dragos Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    1. **Collect Vulnerability**
+    1. **Collect Addresses**
+    1. **Collect Zones**
+    1. **Collect Assets**
+1. **Processing Rules**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:77]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+| config | JSON Object | [Configuration object](#configuration-object) | Yes | Source type specific values. |
+
+### Configuration Object
+
+| Parameter | Type | Required | Default | Description | Example |
+|:--|:--|:--|:--|:--|:--|
+| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
+| description | String | No | `null` | Type a description of the source. | `"Testing source"` |
+| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"` |
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the collector or source. Use the boolean field _siemForward to enable forwarding to SIEM.| `{"_siemForward": false, "fieldA": "valueA"}` |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:92]:
```suggestion
| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the collector or source. Use the boolean field `_siemForward` to enable forwarding to SIEM.| `{"_siemForward": false, "fieldA": "valueA"}` |
```

**Review by jpipkin1 — APPROVED** [review]
This PR looks great, @amee-sumo! I have only a few suggestions.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+#### Endpoint URL
+
+You can use the Hostname to create the Endpoint URL. For example, `https://<hostname>/`.
+
+#### API ID and API Secret
+
+Follow the instructions below to generate the API ID and API Secret:
+
+1. Open the Dragos platform and navigate to the **Admin** > **User**.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:45]:
```suggestion
1. Sign in to the Dragos platform.
1.  Navigate to the **Admin** > **User**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **Dragos**.
+1. Enter a **Name** for the Source. The description is optional.
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema is ignored, known as dropped.
+1. Enter the **Endpoint URL** of Dragos platform. For example, `https://test.cxc.dragos.cloud/`.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:63]:
```suggestion
1. **Endpoint URL**. Enter the Dragos platform endpoint URL. For example, `https://test.cxc.dragos.cloud/`.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+1. Enter the following details for authorization:
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:65]:
```suggestion
1. **API ID**. Enter the API ID of your account collected from the [Dragos platform](#vendor-configuration). For example, `036fxxxx-b642-xxxx-99d3-fcxxxx2exxxx`.
1. **API Secret**. Enter the API Secret of your account collected from the [Dragos platform](#vendor-configuration).. For example, `xxxU1TxxxxxxxxKSJwHYOpK37xxxxxxxxrEHAkU91xxxxxxxxxFrrJ06xxx`.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+    1. **API ID**. API ID of your account. For example, `036fxxxx-b642-xxxx-99d3-fcxxxx2exxxx`.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:66]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+    1. **API Secret**. API Secret of your account. For example, `xxxU1TxxxxxxxxKSJwHYOpK37xxxxxxxxrEHAkU91xxxxxxxxxFrrJ06xxx`.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:67]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+1. Check the following boxes to collect the required data:
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:68]:
```suggestion
1. Select the checkbox depending on the type of data that you wish to collect.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+    1. **Collect Vulnerability**
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:69]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+    1. **Collect Addresses**
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:70]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+    1. **Collect Zones**
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:71]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+    1. **Collect Assets**
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source.md:72]:
```suggestion
```

File: sidebars.ts [hunk 15]
```diff
@@ -424,7 +424,7 @@ module.exports = {
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/cybereason-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/digital-guardian-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source',
-                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source',
+                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source',
```
Comment (JV0812) [inline sidebars.ts:427]:
```suggestion
                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/dropbox-source',
                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/dragos-source',
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by yleiferman** [comment]
Dragos 1.0.0 is being GAed today

**Pull request merged**